modules/nf-core/vardictjava/main.nf

```diff
@@ -40,6 +40,7 @@ process VARDICTJAVA {
     | ${convert_to_vcf} \\
         ${args2} \\
     > ${prefix}.vcf
+    bgzip ${prefix}.vcf
```

Comment:
```suggestion
    | bgzip --threads ${task.cpus} ${prefix}.vcf
```

This way you don't first write the unzipped file. Can you also add an `args2` to bgzip?